### ISSUE TITLE
Bug: build doesn't return correct status code

### ISSUE THREAD

**Issue by C0DK** [Rudi9719/kbtui#11]
When testing i'd like to run `go run build.go build && ./kbtui` - problem is that build.go build doesn't return the correct status code pr unix standards - so my terminal, even if build fails, thinks that the build succeeded, starting up kbtui - which is not ideal, as it then runs an obsolete version

**Comment by Rudi9719** [comment]
Not sure how this would be fixed at our level, might be a bug in https://github.com/magefile/mage ?

**Comment by haukened** [comment]
This should be fixed in PR #22 
Here's the build output after this modification:
```
david@corrin:~/github/kbtui$ go run build.go buildBeta
# _/home/david/github/kbtui
./main.go:7:2: imported and not used: "regexp"
running "go build -tags allcommands,showreactionscmd,emojiList,tabcompletion" failed with exit code 2
exit status 1
david@corrin:~/github/kbtui$ echo $?
1
```

**Comment by Rudi9719** [comment]
Solution in PR #22 merged into dev
